src/rgw/driver/d4n/rgw_sal_d4n.cc

```diff
@@ -738,12 +768,15 @@ int D4NFilterObject::D4NFilterReadOp::D4NFilterGetCB::handle_data(bufferlist& bl
         block.blockID = ofs;
         block.size = bl.length();
         block.version = version;
+        block.dirty = false; //writing to the backend
```

Comment:
why is the dirty flag false here? we are actually writing data to he cache in `handle_data`